src/interfaces/capnp/test/foo.cpp

```diff
@@ -0,0 +1,24 @@
+#include <interfaces/capnp/test/foo.h>
+
+#include <util.h>
+
+namespace interfaces {
+namespace capnp {
+namespace test {
+namespace {
+
+class FooImpl : public FooInterface
+{
+public:
+    int add(int a, int b) override { return a + b; }
+    int mapSize(const std::map<std::string, std::string>& map) override { return map.size(); }
```

Comment:
Not used?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/10102#discussion_r230427589

> Not used?

No, it's called by generated code.